superchain/configs/sepolia-seal-0/sealchain-2.yaml

```diff
@@ -0,0 +1,22 @@
+name: sealchain-2
+chain_id: 6619
+public_rpc: https://rpc-seal-rollup-base-rgf4y72ymx.t.conduit.xyz
+sequencer_rpc: http://awe.some.seq.rpc
```

Comment:
This is from the `.env.example`: `SEQUENCER_RPC="http://awe.some.seq.rpc"`. I don't know if that was intended, but I imagine this should be filled out.

Reply:
Better to leave this blank than to use the example version. 

Without a public RPC, we won't be able to run all of the validation checks. I think this is probably OK for our needs here, though. There is a precedent for not setting this in the `sepolia-dev-0` superchain.

Reply:
will update to leave blank, but we do also now have public sequencer urls for these so i can include them if it doesn't hurt

Reply:
Sure, won't hurt to include proper ones.